packages/vite-plugin-cloudflare/README.md

```diff
@@ -374,6 +374,10 @@ It accepts an optional `PluginConfig` parameter.
   All requests are routed through your entry Worker.
   During the build, each Worker is output to a separate subdirectory of `dist`.
 
+- `inspectorPort?: number | false`
+
+  Optional inspector port to use for debugging your workers, for more details on debugging see the [devtools section](#devtools). Can be set to `false` to disable the debugging inspector altogether. Defaults to `9229`.
```

Comment:
I'm purposely not mentioning `0` as an option here as I've noticed that that is probably a bit buggy right now, I'll look into that in a follow up PR (if that's ok by you) and once it works correctly I'll update the text here.